### PR TITLE
S3 quota check

### PR DESCRIPTION
**Pull request by hardillb**
Only count files if there are files

### CODE REVIEW

File: forge/drivers/s3.js [hunk 1]
```diff
@@ -131,9 +131,11 @@ module.exports = function (app) {
             }))
 
             let size = 0
-            objects.Contents.forEach(file => {
-                size += file.Size
-            })
+            if (objects.Contents) {
+                objects.Contents.forEach(file => {
```
Comment (ZJvandeWeg) [inline forge/drivers/s3.js:135]: Isn't 'reduce()' better in this context? And isn't this missing tests?

**Comment by hardillb** [comment]
Tests to follow, it will need to setup minio in GH Actions as a container. I have most of it, but provisioning the user.

Will add them after release

File: forge/drivers/s3.js (hunk 1 above)
Comment (hardillb) [reply forge/drivers/s3.js:135]: Possibly, I just added the gate round what was already there at this point. I will look at it when I add the tests.

**Review by ZJvandeWeg — APPROVED** [review]

File: forge/drivers/s3.js (hunk 1 above)
Comment (ZJvandeWeg) [reply forge/drivers/s3.js:135]: I'd be fine with stubbing most of it, just doing unit tests

**Pull request merged**